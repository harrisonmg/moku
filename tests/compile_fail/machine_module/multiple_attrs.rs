use moku::*;

#[state_machine]
mod blinky {
    use moku::*;

    #[machine_module]
    #[machine_module]
    mod machine {}
}

fn main() {}
