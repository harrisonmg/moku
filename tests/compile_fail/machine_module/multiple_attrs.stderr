error: multiple `moku::machine_module` attributes defined for this module
 --> tests/compile_fail/machine_module/multiple_attrs.rs:7:5
  |
7 |     #[machine_module]
  |     ^
